python/lsst/meas/algorithms/psfDeterminer.py

```diff
@@ -63,28 +63,40 @@ class BasePsfDeterminerTask(pipeBase.Task, metaclass=abc.ABCMeta):
     _DefaultName = "psfDeterminer"
 
     def __init__(self, config, schema=None, **kwds):
-        """Construct a PSF Determiner
-
-        @param[in]       config   an instance of pexConfig.Config that configures this algorithm
-        @param[in,out]   schema   an instance of afw.table.Schema used for sources; passing a
-                                  schema allows the determiner to reserve a flag field to mark stars
-                                  used in PSF measurement, but some PSF determiners ignore this argument
+        """Construct a PSF Determiner.
+
+        Parameters
+        ----------
+        config : `lsst.pexConfig.Config`
+            Input for configuring the algorithm.
+        schema : `lsst.afw.table.Schema`
+            Schema used for sources; passing a schema allows the determiner
+            to reserve a flag field to mark stars used in PSF measurement,
+            but some PSF determiners ignore this argument.
```

Comment:
Oh, I forgot: the `__init__` parameters should go in the class docstring, and `__init__` has no docstring.

https://developer.lsst.io/python/numpydoc.html#placement-of-class-docstrings